config/main.py

```diff
@@ -3674,8 +3678,12 @@ def neighbor(ipaddr_or_hostname, verbose):
         if _change_bgp_session_status(config_db, ipaddr_or_hostname, 'up', verbose):
             found_neighbor = True
 
+    device_metadata = config_db.get_entry('DEVICE_METADATA', 'localhost')
     if not found_neighbor:
-        click.get_current_context().fail("Could not locate neighbor '{}'".format(ipaddr_or_hostname))
+        if device_metadata['type'] == 'SpineRouter':
```

Comment:
Same concern on this check may allow T1 failure unchecked as T1 is also "SpineRouter".

Reply:
never mind.. got clarified with @abdosi T1 is "LeafRouter" so we are ok with this check.